js/request.js

```diff
@@ -64,19 +63,24 @@ var startJob = function (job, requestObject, handler, opts) {
   var event = requestObject.event;
 
   var context = makeContextObject(opts.timeout, requestObject.context || {});
-  context.done = function (err, result) {
-    job.doCompletion(err, result);
-    if (err) {
-      console.warn('Error:', err);
-    }
+
+  const done = (err, res) => {
+      if (err) {
```

Comment:
You've got a mixture of two and four space indentation in this file.

Reply:
👍 sorry, should be fixed now, I've adapted to 2 space indentation since that was what was used in the existing code.